packages/engine-core/src/common/Engine.ts

```diff
@@ -63,7 +63,7 @@ export abstract class Engine<InteractiveTransactionPayload = unknown> {
   abstract on(event: EngineEventType, listener: (args?: any) => any): void
   abstract start(): Promise<void>
   abstract stop(): Promise<void>
-  abstract getDmmf(): Promise<DMMF.Document>
+  abstract getDmmf(traceparent?: string): Promise<DMMF.Document>
```

Comment:
Could also retrieve it directly in the engine instead of passing it?

Reply:
Done!